### ISSUE TITLE
make_apply error 

### ISSUE THREAD

**Issue by snassimr** [GoogleCloudPlatform/kubeflow-distribution#413]
I am gettin the following error 

asmcli: [ERROR]: Autopilot clusters are only supported with managed control plane.

I follow this https://googlecloudplatform.github.io/kubeflow-gke-docs/docs/deploy/deploy-cli/ for Kubeflow 1.6 install

Any solution for this ?

**Comment by itsazzad** [comment]
I am also getting the same error.
```
~/kubeflow-distribution/kubeflow 130 » make apply
write /dev/stdout: broken pipe
cluster_name_regex=^[a-z][-a-z0-9]{0,22}[a-z0-9]$
The kubeflow cluster name "kubeflow" is valid.
PROJECT=kubeflow-380816 NAME=kubeflow ./hack/check_domain_length.sh
Build directory: ./build
Component path: common/managed-storage
Apply component resources: common/managed-storage
Found Makefile, call 'make apply' of this component Makefile.
make[1]: Entering directory '/home/sazzad/kubeflow-distribution/kubeflow/common/managed-storage'
rm -rf ./build
mkdir -p ./build
kustomize build -o ./build/ .
kubectl --context=kubeflow-mgmt apply -f ./build
sqlinstance.sql.cnrm.cloud.google.com/kubeflow-kfp unchanged
storagebucket.storage.cnrm.cloud.google.com/kubeflow-380816-kfp unchanged
# Wait for all Google Cloud resources to get created and become ready.
# If this takes long, you can view status by:
cd common/managed-storage && make status
# For resources with READY=False, debug by:
kubectl --context=kubeflow-mgmt -n kubeflow-380816 describe <KIND>/<NAME>

kubectl --context=kubeflow-mgmt wait --for=condition=Ready --timeout=100s -f ./build \
        || kubectl --context=kubeflow-mgmt get -f ./build
sqlinstance.sql.cnrm.cloud.google.com/kubeflow-kfp condition met
storagebucket.storage.cnrm.cloud.google.com/kubeflow-380816-kfp condition met
kubectl --context=kubeflow-mgmt wait --for=condition=Ready --timeout=500s -f ./build
sqlinstance.sql.cnrm.cloud.google.com/kubeflow-kfp condition met
storagebucket.storage.cnrm.cloud.google.com/kubeflow-380816-kfp condition met
make[1]: Leaving directory '/home/sazzad/kubeflow-distribution/kubeflow/common/managed-storage'
Build directory: ./build
Component path: common/cnrm
Apply component resources: common/cnrm
Found Makefile, call 'make apply' of this component Makefile.
make[1]: Entering directory '/home/sazzad/kubeflow-distribution/kubeflow/common/cnrm'
echo ./build
./build
make apply-cnrm build_dir=./build
make[2]: Entering directory '/home/sazzad/kubeflow-distribution/kubeflow/common/cnrm'
rm -rf ./build && mkdir -p ./build
kustomize build -o ./build ./
kubectl --context=kubeflow-mgmt apply -f ./build
computeaddress.compute.cnrm.cloud.google.com/kubeflow-ip unchanged
Warning: metadata.clusterName was specified. This field is not preserved and will be removed from the schema in 1.25
containercluster.container.cnrm.cloud.google.com/kubeflow configured
iampolicymember.iam.cnrm.cloud.google.com/kubeflow-admin-bigquery unchanged
iampolicymember.iam.cnrm.cloud.google.com/kubeflow-admin-cloudbuild unchanged
iampolicymember.iam.cnrm.cloud.google.com/kubeflow-admin-cloudsql unchanged
iampolicymember.iam.cnrm.cloud.google.com/kubeflow-admin-dataflow unchanged
iampolicymember.iam.cnrm.cloud.google.com/kubeflow-admin-dataproc unchanged
iampolicymember.iam.cnrm.cloud.google.com/kubeflow-admin-istio-wi unchanged
iampolicymember.iam.cnrm.cloud.google.com/kubeflow-admin-kubeflow-wi unchanged
iampolicymember.iam.cnrm.cloud.google.com/kubeflow-admin-logging unchanged
iampolicymember.iam.cnrm.cloud.google.com/kubeflow-admin-manages-user unchanged
iampolicymember.iam.cnrm.cloud.google.com/kubeflow-admin-metricwriter unchanged
iampolicymember.iam.cnrm.cloud.google.com/kubeflow-admin-ml unchanged
iampolicymember.iam.cnrm.cloud.google.com/kubeflow-admin-monitoringviewer unchanged
iampolicymember.iam.cnrm.cloud.google.com/kubeflow-admin-network unchanged
iampolicymember.iam.cnrm.cloud.google.com/kubeflow-admin-servicemanagement unchanged
iampolicymember.iam.cnrm.cloud.google.com/kubeflow-admin-source unchanged
iampolicymember.iam.cnrm.cloud.google.com/kubeflow-admin-storage unchanged
iampolicymember.iam.cnrm.cloud.google.com/kubeflow-admin-viewer unchanged
iampolicymember.iam.cnrm.cloud.google.com/kubeflow-admin-workload-identity-user unchanged
iampolicymember.iam.cnrm.cloud.google.com/kubeflow-user-bigquery unchanged
iampolicymember.iam.cnrm.cloud.google.com/kubeflow-user-cloudbuild unchanged
iampolicymember.iam.cnrm.cloud.google.com/kubeflow-user-cloudsql unchanged
iampolicymember.iam.cnrm.cloud.google.com/kubeflow-user-dataflow unchanged
iampolicymember.iam.cnrm.cloud.google.com/kubeflow-user-dataproc unchanged
iampolicymember.iam.cnrm.cloud.google.com/kubeflow-user-logging unchanged
iampolicymember.iam.cnrm.cloud.google.com/kubeflow-user-metricwriter unchanged
iampolicymember.iam.cnrm.cloud.google.com/kubeflow-user-ml unchanged
iampolicymember.iam.cnrm.cloud.google.com/kubeflow-user-monitoringviewer unchanged
iampolicymember.iam.cnrm.cloud.google.com/kubeflow-user-source unchanged
iampolicymember.iam.cnrm.cloud.google.com/kubeflow-user-storage unchanged
iampolicymember.iam.cnrm.cloud.google.com/kubeflow-user-viewer unchanged
iampolicymember.iam.cnrm.cloud.google.com/kubeflow-user-workload-identity-user-ml-pipeline-ui unchanged
iampolicymember.iam.cnrm.cloud.google.com/kubeflow-user-workload-identity-user-ml-pipeline-visualizationserver unchanged
iampolicymember.iam.cnrm.cloud.google.com/kubeflow-user-workload-identity-user-pipeline-runner unchanged
iampolicymember.iam.cnrm.cloud.google.com/kubeflow-vm-logging unchanged
iampolicymember.iam.cnrm.cloud.google.com/kubeflow-vm-policy-cloudtrace unchanged
iampolicymember.iam.cnrm.cloud.google.com/kubeflow-vm-policy-meshtelemetry unchanged
iampolicymember.iam.cnrm.cloud.google.com/kubeflow-vm-policy-monitoring-viewer unchanged
iampolicymember.iam.cnrm.cloud.google.com/kubeflow-vm-policy-monitoring unchanged
iampolicymember.iam.cnrm.cloud.google.com/kubeflow-vm-policy-storage unchanged
iamserviceaccount.iam.cnrm.cloud.google.com/kubeflow-admin unchanged
iamserviceaccount.iam.cnrm.cloud.google.com/kubeflow-user unchanged
iamserviceaccount.iam.cnrm.cloud.google.com/kubeflow-vm unchanged
service.serviceusage.cnrm.cloud.google.com/anthos.googleapis.com unchanged
service.serviceusage.cnrm.cloud.google.com/cloudbuild.googleapis.com unchanged
service.serviceusage.cnrm.cloud.google.com/cloudresourcemanager.googleapis.com unchanged
service.serviceusage.cnrm.cloud.google.com/compute.googleapis.com unchanged
service.serviceusage.cnrm.cloud.google.com/container.googleapis.com unchanged
service.serviceusage.cnrm.cloud.google.com/gkeconnect.googleapis.com unchanged
service.serviceusage.cnrm.cloud.google.com/gkehub.googleapis.com unchanged
service.serviceusage.cnrm.cloud.google.com/iamcredentials.googleapis.com unchanged
service.serviceusage.cnrm.cloud.google.com/iap.googleapis.com unchanged
service.serviceusage.cnrm.cloud.google.com/logging.googleapis.com unchanged
service.serviceusage.cnrm.cloud.google.com/meshca.googleapis.com unchanged
service.serviceusage.cnrm.cloud.google.com/meshconfig.googleapis.com unchanged
service.serviceusage.cnrm.cloud.google.com/meshtelemetry.googleapis.com unchanged
service.serviceusage.cnrm.cloud.google.com/monitoring.googleapis.com unchanged
service.serviceusage.cnrm.cloud.google.com/servicemanagement.googleapis.com unchanged
make[2]: Leaving directory '/home/sazzad/kubeflow-distribution/kubeflow/common/cnrm'
make wait-gcp
make[2]: Entering directory '/home/sazzad/kubeflow-distribution/kubeflow/common/cnrm'
# Wait for all Google Cloud resources to get created and become ready.
Waiting for iamserviceaccount resources...
iamserviceaccount.iam.cnrm.cloud.google.com/kubeflow-admin condition met
iamserviceaccount.iam.cnrm.cloud.google.com/kubeflow-user condition met
iamserviceaccount.iam.cnrm.cloud.google.com/kubeflow-vm condition met
Waiting for iampolicymember resources...
iampolicymember.iam.cnrm.cloud.google.com/kubeflow-admin-bigquery condition met
iampolicymember.iam.cnrm.cloud.google.com/kubeflow-admin-cloudbuild condition met
iampolicymember.iam.cnrm.cloud.google.com/kubeflow-admin-cloudsql condition met
iampolicymember.iam.cnrm.cloud.google.com/kubeflow-admin-dataflow condition met
iampolicymember.iam.cnrm.cloud.google.com/kubeflow-admin-dataproc condition met
iampolicymember.iam.cnrm.cloud.google.com/kubeflow-admin-istio-wi condition met
iampolicymember.iam.cnrm.cloud.google.com/kubeflow-admin-kubeflow-wi condition met
iampolicymember.iam.cnrm.cloud.google.com/kubeflow-admin-logging condition met
iampolicymember.iam.cnrm.cloud.google.com/kubeflow-admin-manages-user condition met
iampolicymember.iam.cnrm.cloud.google.com/kubeflow-admin-metricwriter condition met
iampolicymember.iam.cnrm.cloud.google.com/kubeflow-admin-ml condition met
iampolicymember.iam.cnrm.cloud.google.com/kubeflow-admin-monitoringviewer condition met
iampolicymember.iam.cnrm.cloud.google.com/kubeflow-admin-network condition met
iampolicymember.iam.cnrm.cloud.google.com/kubeflow-admin-servicemanagement condition met
iampolicymember.iam.cnrm.cloud.google.com/kubeflow-admin-source condition met
iampolicymember.iam.cnrm.cloud.google.com/kubeflow-admin-storage condition met
iampolicymember.iam.cnrm.cloud.google.com/kubeflow-admin-viewer condition met
iampolicymember.iam.cnrm.cloud.google.com/kubeflow-admin-workload-identity-user condition met
iampolicymember.iam.cnrm.cloud.google.com/kubeflow-user-bigquery condition met
iampolicymember.iam.cnrm.cloud.google.com/kubeflow-user-cloudbuild condition met
iampolicymember.iam.cnrm.cloud.google.com/kubeflow-user-cloudsql condition met
iampolicymember.iam.cnrm.cloud.google.com/kubeflow-user-dataflow condition met
iampolicymember.iam.cnrm.cloud.google.com/kubeflow-user-dataproc condition met
iampolicymember.iam.cnrm.cloud.google.com/kubeflow-user-logging condition met
iampolicymember.iam.cnrm.cloud.google.com/kubeflow-user-metricwriter condition met
iampolicymember.iam.cnrm.cloud.google.com/kubeflow-user-ml condition met
iampolicymember.iam.cnrm.cloud.google.com/kubeflow-user-monitoringviewer condition met
iampolicymember.iam.cnrm.cloud.google.com/kubeflow-user-source condition met
iampolicymember.iam.cnrm.cloud.google.com/kubeflow-user-storage condition met
iampolicymember.iam.cnrm.cloud.google.com/kubeflow-user-viewer condition met
iampolicymember.iam.cnrm.cloud.google.com/kubeflow-user-workload-identity-user-ml-pipeline-ui condition met
iampolicymember.iam.cnrm.cloud.google.com/kubeflow-user-workload-identity-user-ml-pipeline-visualizationserver condition met
iampolicymember.iam.cnrm.cloud.google.com/kubeflow-user-workload-identity-user-pipeline-runner condition met
iampolicymember.iam.cnrm.cloud.google.com/kubeflow-vm-logging condition met
iampolicymember.iam.cnrm.cloud.google.com/kubeflow-vm-policy-cloudtrace condition met
iampolicymember.iam.cnrm.cloud.google.com/kubeflow-vm-policy-meshtelemetry condition met
iampolicymember.iam.cnrm.cloud.google.com/kubeflow-vm-policy-monitoring condition met
iampolicymember.iam.cnrm.cloud.google.com/kubeflow-vm-policy-monitoring-viewer condition met
iampolicymember.iam.cnrm.cloud.google.com/kubeflow-vm-policy-storage condition met
Waiting for computeaddress resources...
computeaddress.compute.cnrm.cloud.google.com/kubeflow-ip condition met
Waiting for containercluster resources...
containercluster.container.cnrm.cloud.google.com/kubeflow condition met
make[2]: Leaving directory '/home/sazzad/kubeflow-distribution/kubeflow/common/cnrm'
make create-ctxt
make[2]: Entering directory '/home/sazzad/kubeflow-distribution/kubeflow/common/cnrm'
PROJECT=kubeflow-380816 \
   REGION=us-east1 \
   NAME=kubeflow ../../hack/create_context.sh
+ kubectl config delete-context kubeflow
warning: this removed your active context, use "kubectl config use-context" to select a different one
deleted context kubeflow from /home/sazzad/.kube/config
+ set -ex
+ NAMESPACE=kubeflow
+ gcloud --project=kubeflow-380816 container clusters get-credentials --region=us-east1 kubeflow
Fetching cluster endpoint and auth data.
kubeconfig entry generated for kubeflow.
++ kubectl config current-context
+ kubectl config rename-context gke_kubeflow-380816_us-east1_kubeflow kubeflow
Context "gke_kubeflow-380816_us-east1_kubeflow" renamed to "kubeflow".
+ kubectl config set-context --current --namespace=kubeflow
Context "kubeflow" modified.
make[2]: Leaving directory '/home/sazzad/kubeflow-distribution/kubeflow/common/cnrm'
make[1]: Leaving directory '/home/sazzad/kubeflow-distribution/kubeflow/common/cnrm'
Build directory: ./build
Component path: asm
Apply component resources: asm
Found Makefile, call 'make apply' of this component Makefile.
make[1]: Entering directory '/home/sazzad/kubeflow-distribution/kubeflow/asm'
curl https://storage.googleapis.com/csm-artifacts/asm/asmcli_1.14.1-asm.3-config6 > asmcli;
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  195k  100  195k    0     0   103k      0  0:00:01  0:00:01 --:--:--  102k
chmod +x asmcli
rm -rf asm.tar.gz
curl -LJ https://github.com/GoogleCloudPlatform/anthos-service-mesh-packages/archive/refs/tags/1.14.1-asm.3+config6.tar.gz -o asm.tar.gz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  242k    0  242k    0     0  38493      0 --:--:--  0:00:06 --:--:-- 62635
rm -rf ./package
mkdir ./package
tar -xf asm.tar.gz --strip-components=1 -C ./package
./asmcli install \
--project_id kubeflow-380816 \
--cluster_name kubeflow \
--cluster_location us-east1 \
--output_dir ./package \
--enable_all \
--ca mesh_ca \
--custom_overlay ./package/asm/istio/options/iap-operator.yaml \
--custom_overlay ./options/ingressgateway-iap.yaml \
--option legacy-default-ingressgateway \
--verbose
asmcli: Setting up necessary files...
asmcli: Using /home/sazzad/kubeflow-distribution/kubeflow/asm/package/asm_kubeconfig as the kubeconfig...
asmcli: Checking installation tool dependencies...
asmcli: Fetching/writing GCP credentials to kubeconfig file...
asmcli: Running: '/home/sazzad/google-cloud-sdk/bin/gcloud container clusters get-credentials kubeflow --project=kubeflow-380816 --zone=us-east1'
asmcli: -------------
Fetching cluster endpoint and auth data.
kubeconfig entry generated for kubeflow.
asmcli: Running: '/home/sazzad/google-cloud-sdk/bin/kubectl --kubeconfig /home/sazzad/kubeflow-distribution/kubeflow/asm/package/asm_kubeconfig config current-context'
asmcli: -------------
asmcli: Verifying connectivity (10s)...
asmcli: Running: '/home/sazzad/google-cloud-sdk/bin/kubectl --kubeconfig /home/sazzad/kubeflow-distribution/kubeflow/asm/package/asm_kubeconfig --context gke_kubeflow-380816_us-east1_kubeflow config view --minify=true -ojson'
asmcli: -------------
asmcli: Running: 'nc -zvw 10 34.73.112.179 443'
asmcli: -------------
Connection to 34.73.112.179 443 port [tcp/https] succeeded!
asmcli: kubeconfig set to /home/sazzad/kubeflow-distribution/kubeflow/asm/package/asm_kubeconfig
asmcli: using context gke_kubeflow-380816_us-east1_kubeflow
asmcli: Getting account information...
asmcli: Running: '/home/sazzad/google-cloud-sdk/bin/gcloud auth list --project=kubeflow-380816 --filter=status:ACTIVE --format=value(account)'
asmcli: -------------
asmcli: Running: '/home/sazzad/google-cloud-sdk/bin/gcloud config get-value auth/impersonate_service_account'
asmcli: -------------
(unset)
asmcli: Running: '/home/sazzad/google-cloud-sdk/bin/gcloud container clusters list --project=kubeflow-380816 --filter=name = kubeflow AND location = us-east1 --format=value(name)'
asmcli: -------------
WARNING: --filter : operator evaluation is changing for consistency across Google APIs.  name=kubeflow currently does not match but will match in the near future.  Run `gcloud topic filters` for details.
asmcli: Running: '/home/sazzad/google-cloud-sdk/bin/kpt version'
asmcli: -------------
asmcli: Downloading kpt..
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 11.8M  100 11.8M    0     0   933k      0  0:00:12  0:00:12 --:--:--  885k
asmcli: Downloading ASM..
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 39.7M  100 39.7M    0     0  2072k      0  0:00:19  0:00:19 --:--:-- 1959k
asmcli: Downloading ASM kpt package...
asmcli: Running: '/home/sazzad/kubeflow-distribution/kubeflow/asm/package/kpt pkg get --auto-set=false https://github.com/GoogleCloudPlatform/anthos-service-mesh-packages.git/asm@1.14.1-asm.3+config6 asm'
asmcli: -------------
fetching package "/asm" from "https://github.com/GoogleCloudPlatform/anthos-service-mesh-packages" to "asm/asm"
asmcli: Running: '/home/sazzad/kubeflow-distribution/kubeflow/asm/package/kpt pkg get --auto-set=false https://github.com/GoogleCloudPlatform/anthos-service-mesh-packages.git/samples@1.14.1-asm.3+config6 samples'
asmcli: -------------
fetching package "/samples" from "https://github.com/GoogleCloudPlatform/anthos-service-mesh-packages" to "samples/samples"
asmcli: Verifying cluster registration.
asmcli: Running: '/home/sazzad/google-cloud-sdk/bin/kubectl --kubeconfig /home/sazzad/kubeflow-distribution/kubeflow/asm/package/asm_kubeconfig --context gke_kubeflow-380816_us-east1_kubeflow get memberships.hub.gke.io membership -o=json'
asmcli: -------------
asmcli: Running: '/home/sazzad/google-cloud-sdk/bin/kubectl --kubeconfig /home/sazzad/kubeflow-distribution/kubeflow/asm/package/asm_kubeconfig --context gke_kubeflow-380816_us-east1_kubeflow get memberships.hub.gke.io membership -o=jsonpath={.spec.identity_provider}'
asmcli: -------------
asmcli: Running: '/home/sazzad/google-cloud-sdk/bin/gcloud container hub memberships list --project kubeflow-380816 --format=json'
asmcli: -------------
asmcli: Running: '/home/sazzad/google-cloud-sdk/bin/gcloud projects describe kubeflow-380816 --format value(projectNumber)'
asmcli: -------------
asmcli: Verified cluster is registered to kubeflow-380816
asmcli: Enabling required APIs...
asmcli: Running: '/home/sazzad/google-cloud-sdk/bin/gcloud services enable --project=kubeflow-380816 mesh.googleapis.com'
asmcli: -------------
asmcli: Running: '/home/sazzad/google-cloud-sdk/bin/gcloud container clusters describe --project=kubeflow-380816 --region us-east1 kubeflow --format=json'
asmcli: -------------
asmcli: Running: '/home/sazzad/google-cloud-sdk/bin/gcloud container clusters describe --project=kubeflow-380816 --region us-east1 kubeflow --format=json'
asmcli: -------------
asmcli: Verifying cluster registration.
asmcli: Running: '/home/sazzad/google-cloud-sdk/bin/gcloud container hub memberships list --project kubeflow-380816 --format=json'
asmcli: -------------
asmcli: Running: '/home/sazzad/google-cloud-sdk/bin/gcloud projects describe kubeflow-380816 --format value(projectNumber)'
asmcli: -------------
asmcli: Verified cluster is registered to kubeflow-380816
asmcli: Verifying cluster registration.
asmcli: Running: '/home/sazzad/google-cloud-sdk/bin/gcloud container hub memberships list --project kubeflow-380816 --format=json'
asmcli: -------------
asmcli: Running: '/home/sazzad/google-cloud-sdk/bin/gcloud projects describe kubeflow-380816 --format value(projectNumber)'
asmcli: -------------
asmcli: Verified cluster is registered to kubeflow-380816
asmcli: Checking for project kubeflow-380816...
asmcli: Running: '/home/sazzad/google-cloud-sdk/bin/gcloud projects describe kubeflow-380816 --format=value(projectNumber)'
asmcli: -------------
asmcli: Reading labels for us-east1/kubeflow...
asmcli: Running: '/home/sazzad/google-cloud-sdk/bin/gcloud container clusters describe kubeflow --zone=us-east1 --project=kubeflow-380816 --format=value(resourceLabels)[delimiter=","]'
asmcli: -------------
asmcli: Querying for core/account...
asmcli: Running: '/home/sazzad/google-cloud-sdk/bin/gcloud config get-value core/account'
asmcli: -------------
asmcli: Binding itsazzad@gmail.com to cluster admin role...
asmcli: Running: '/home/sazzad/google-cloud-sdk/bin/kubectl --kubeconfig /home/sazzad/kubeflow-distribution/kubeflow/asm/package/asm_kubeconfig --context gke_kubeflow-380816_us-east1_kubeflow create clusterrolebinding itsazzad-cluster-admin-binding --clusterrole=cluster-admin --user=itsazzad@gmail.com --dry-run=client -o yaml'
asmcli: -------------
asmcli: Running: '/home/sazzad/google-cloud-sdk/bin/kubectl --kubeconfig /home/sazzad/kubeflow-distribution/kubeflow/asm/package/asm_kubeconfig --context gke_kubeflow-380816_us-east1_kubeflow apply -f -'
asmcli: -------------
clusterrolebinding.rbac.authorization.k8s.io/itsazzad-cluster-admin-binding configured
asmcli: Creating istio-system namespace...
asmcli: Running: '/home/sazzad/google-cloud-sdk/bin/kubectl --kubeconfig /home/sazzad/kubeflow-distribution/kubeflow/asm/package/asm_kubeconfig --context gke_kubeflow-380816_us-east1_kubeflow get ns'
asmcli: -------------
asmcli: [ERROR]: Autopilot clusters are only supported with managed control plane.
make[1]: *** [Makefile:42: install-asm] Error 2
make[1]: Leaving directory '/home/sazzad/kubeflow-distribution/kubeflow/asm'
make: *** [Makefile:83: apply] Error 1
```

**Comment by zijianjoy** [comment]
cc @gkcalat

**Comment by gkcalat** [comment]
Thank you for reporting this @itsazzad and @snassimr!

We have released a [pre-release of Kubeflow 1.7](https://github.com/GoogleCloudPlatform/kubeflow-distribution/releases/tag/v1.7.0-rc.0). It has an upgraded ASM and should resolve your issue. The corresponding tag is `v1.7.0-rc.0`.

Feel free to reopen this issue if the problem persists.

**Comment by gkcalat** [comment]
Kubeflow on Google Cloud v1.7.0 [has been released](https://github.com/GoogleCloudPlatform/kubeflow-distribution/releases/tag/v1.7.0). Please, upgrade your deployment.
